src/main/scala/firrtl/Emitter.scala

```diff
@@ -454,7 +454,11 @@ class VerilogEmitter extends Emitter with PassBased {
           instdeclares += Seq(");")
           sx
         case sx: DefMemory =>
-          declare("reg", sx.name, VectorType(sx.dataType, sx.depth))
+          val fullSize = sx.depth*(sx.dataType match { case GroundType(i:IntWidth) => i.width })
```

Comment:
Space around `*` and please unapply `IntWidth` as well (ie. ` case GroundType(IntWidth(width)) => width`)